src/index.html

```diff
@@ -10,10 +10,25 @@
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap&subset=cyrillic" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
+
 </head>
 <body>
-
+  <header class="header">
+    <nav class="nav">
+      <ul class="nav_list">
+        <li class="nav_item"><a class="link is-active" href="#1">APPLE</a></li>
```

Comment:
![image](https://user-images.githubusercontent.com/44294945/64355149-61774400-d009-11e9-8ad8-55d166eddf19.png)

remove this margin